horace_core/sqw/@sqw/private/cut_sqw_sym_main_single.m

```diff
@@ -80,12 +80,8 @@
         % This could be generalized later - but with repercussions in many routines
         header_ave=header_average(header);
         
-        
-        % Because Alex's aProjectionBase class (of which proj is an instance)
-        % is a rather odd hybrid object with projection and cut information
-        % in its public and private properties, regenerate this class
-        % with the information for input cut projection information in the
-        % same way that was done by cut_sqw_check_pbins in cut_sqw_sym_main
+        % 
+        %TODO: THIS ALL SHOULD GO. Modify this as its now done in cut_sqw_sym_main
```

Comment:
What should all go? All this file? All this method? 

It would be good to say if the deleted comment has been dealt with, if not it should be retained

Reply:
hopefully done